Makefile

```diff
@@ -14,8 +14,7 @@ docs:
 	mkdocs serve
 
 pypi:
-	python setup.py sdist
-	python setup.py bdist_wheel --universal
+	python -m build
```

Comment:
I have overseen this command in the Makefile before.

The pypi command used the deleted setup.py file, which lead to an error.

I added `python -m build` instead, which uses the new recommended [build](https://build.pypa.io) tool.

Reply:
If I'm not mistaken, using `build` would require a new [pipeline](https://carpentries-incubator.github.io/python_packaging/instructor/05-publishing.html#build-and-twine) which complicates my process to push it to pypi. Instead, I would suggest following the [scikit-lego](https://github.com/koaning/scikit-lego/blob/main/setup.py) approach such that the original pipeline would remain intact.

Reply:
I'm sorry, but what exactly do you mean by pipeline?
In my opinion the result of the two replaced lines should be identical to running python -m build. Running `make pypi` should build the sdist and wheel in `dist`, which gets picked up by twine.
I don't think the setup.py is needed any longer. But sure, if you prefer to keep it, the minimal one from your link should be fine.

What do you think?

Reply:
You know what, let's indeed remove `setup.py` but keep setuptools as the build system. That removes an additional unnecessary file but still allows me to keep the familiarity I have with setuptools. 

Makes debugging for me easier when I do stuff with pypi but still allows for the newer format. Does that make sense?